pkg/webui/console/containers/side-bar/navigation/general-side-navigation.js

```diff
@@ -55,7 +55,7 @@ const GeneralSideNavigation = () => {
         )}
         <SideNavigation.Item
           title={sharedMessages.notifications}
-          path="/notifications"
+          path="/notifications/inbox"
```

Comment:
```suggestion
          path="/notifications"
```
I realize this has to be `/notifications` only, otherwise `/notifications/archive` will not match and the root route will forward to `/notifications/inbox` anyway.